packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -156,10 +162,15 @@ export default function Sidebar({
     optOutWeeklyGoal,
   } = useContext(SettingsContext);
   const [showSettings, setShowSettings] = useState(false);
+  const [showRecommendAnArticle, setShowRecommendAnArticle] = useState(false);
```

Comment:
This is just a general comment on the term that we're gonna be using. I think it would be better to use `suggest` than `recommend` as the latter might get used in the future like in a "Recommendation" feature where we recommend articles to read. I feel like the current term is bringing a different meaning (actually got a bit confused at first). Also that we use `suggest` in our backend terms.

```suggestion
  const [showSuggestArticle, setShowSuggestArticle] = useState(false);
```

Reply:
Not sure, the whole system is called recommend an article everywhere.
Perhaps we should raise it with product/graham.

Reply:
I agree. We should coordinate this with the product team.